packages/react/src/components/CalloutData/__stories__/CalloutData.stories.js

```diff
@@ -40,11 +40,24 @@ export default !DDS_CALLOUT_DATA
 export const Default = !DDS_CALLOUT_DATA
   ? undefined
   : ({ parameters }) => {
+      let data;
+      if (parameters.props.CalloutData.props.data.includes('%20')) {
+        data = parameters.props.CalloutData.props.data
+          .split('%20')
+          .map(elem => `${elem} `)
+          .join('');
+      } else {
+        data = parameters.props.CalloutData.props.data;
+      }
+      console.log();
```

Comment:
Make sure to remove the `console.log()`

Reply:
Removed that in my latest commit, thanks!